### DISCUSSION TITLE
Window ETA?

### DISCUSSION THREAD

**Discussion by ketpain** [zed-industries/zed#15392] [Feature Requests]
Is there a windows release ETA or just kinda when it happens, it happens? I saw a question posted back in February. Any changes?

(I'm also aware that I can build it on windows, but a stable official release is what I want)

This editor looks fantastic!!

**Comment by SomeoneToIgnore** [comment]
> when it happens, it happens

This.
Currently, no plans to involve Zed team into another platform endeavor, as Linux has some issues to fix first: https://github.com/zed-industries/zed/issues?q=is%3Aopen+is%3Aissue+label%3Alinux+

I cannot know this for sure, but so far it feels that the "official" Windows work is not happening this year.
